### PR TITLE
adding ORM pool configuration

### LINKED ISSUE #518: Expose database connection pool configuration
When I start following peertube account from Mastodon instance my peertube instance throws this error 

error: Cannot create remote actor and check signature. {
  "err": "TimeoutError: ResourceRequest timed out\n    at ResourceRequest._fireTimeout (/var/www/peertube/versions/peertube-v1.0.0-beta.3/node_modules/generic-pool/lib/ResourceRequest.js:62:17)\n    at Timeout.bound (/var/www/peertube/versions/peertube-v1.0.0-beta.3/node_modules/generic-pool/lib/ResourceRequest.js:8:15)\n    at ontimeout (timers.js:482:11)\n    at tryOnTimeout (timers.js:317:5)\n    at Timer.listOnTimeout (timers.js:277:5)"

* **PeerTube version or URL:**
PeerTube v1.0.0-beta.3
* **Browser name/version:**
Safari 11.1
* **NodeJS version:**
v18.11.1

* **What is the expected behaviour?**
Expected behaviour is be able to follow peertube instance's account from Mastodon

* **What do you see instead?**
Mentioned above error.

Thanks to point me to the right direction to solve this.

### PR DESCRIPTION
**Pull request by rigelk**
Resolves #518

### CODE REVIEW

File: config/default.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ database:
   suffix: '_dev'
   username: 'peertube'
   password: 'peertube'
+  pool: 5
```
Comment (Chocobozzz) [inline config/default.yaml]: `pool.max` or `poolMax` would be a better name

File: server/initializers/database.ts [hunk 2]
```diff
@@ -40,6 +41,11 @@ const sequelizeTypescript = new SequelizeTypescript({
   port,
   username,
   password,
+  pool: {
+    max: poolMax,
+    idle: 30000,
+    acquire: 60000
```
Comment (Chocobozzz) [inline server/initializers/database.ts]: Where those values come from?

Comment (rigelk) [reply server/initializers/database.ts]: You're right, I should use the default pooling config: https://github.com/sequelize/sequelize/blob/63d4aaa3298e7a8a9b12d96deef1b8d015af712a/lib/dialects/abstract/connection-manager.js#L10-L17

**Comment by Chocobozzz** [comment]
Thanks!

**Pull request merged**